java/fory-core/src/main/java/org/apache/fory/type/DescriptorGrouper.java

```diff
@@ -178,6 +178,7 @@ private DescriptorGrouper(
       } else if (TypeUtils.isMap(descriptor.getRawType())) {
         mapDescriptors.add(descriptorUpdater.apply(descriptor));
       } else if (isMonomorphic.test(descriptor.getRawType())) {
+        isMonomorphic.test(descriptor.getRawType());
```

Comment:
I am totally new here, this duplication just looks strange. If there is a reason for, it, A comment with explanation would be great :)

Reply:
I just removed it, it's added by me for debugging, thanks for catching this